### ISSUE TITLE
Error message while/after files copied

### ISSUE THREAD

**Issue by solidevolution** [sapphi-red/vite-plugin-static-copy#65]
I need to copy some assets from one package in my npm workspace directory to other packages in workspace.

I use the following vite config to build my library. 

```
import { resolve } from 'path'

import { viteStaticCopy } from 'vite-plugin-static-copy'
import { defineConfig } from 'vite'
import vue from '@vitejs/plugin-vue'

export default defineConfig({
  plugins: [
    viteStaticCopy({
      targets: [
        {
          src: './public/shared',
          dest: './../../app/public',
        },
      ],
    }),
    vue(),
  ] as any[],
  build: {
    lib: {
      entry: resolve(__dirname, 'src/lib.ts'),
      name: 'Name',
      fileName: 'index',
    },
    rollupOptions: {
      external: ['vue'],
      output: {
        globals: {
          vue: 'Vue',
        },
      },
    },
  },
})
```

Then if I watch (or build) the library, I get this error (but the files are copied correctly)

```
> vue-tsc && vite build --watch

vite v5.0.1 building for production...

watching for file changes...

build started...
✓ 6 modules transformed.
dist/index.umd.cjs  0.88 kB │ gzip: 0.57 kB
dist/style.css  8.86 kB │ gzip: 2.01 kB
dist/index.js   0.85 kB │ gzip: 0.52 kB
[vite-plugin-static-copy:build] ENOENT: no such file or directory, unlink '/packages/app/public/shared/fonts/Inter-Bold.woff2'
[vite-plugin-static-copy] Copied 1 items.
```

**Comment by sapphi-red** [comment]
Would you create a github repository?

**Comment by solidevolution** [comment]
I will do tomorrow, currently there are some client specific names in it.
If you give me your email I can send you an email.

**Comment by solidevolution** [comment]
@sapphi-red here an example monorepo with the problem: https://github.com/solidevolution/vite-plugin-static-copy-issue/tree/main

**Comment by sapphi-red** [comment]
I have released a fix for this (v1.0.0).
